tools/web-test-runner-config.js

```diff
@@ -0,0 +1,114 @@
+import { argv } from 'node:process';
+import { defaultReporter } from '@web/test-runner';
+import { playwrightLauncher } from '@web/test-runner-playwright';
+import { visualDiff } from './visual-diff-plugin.js';
+import { visualDiffReporter } from './visual-diff-reporter.js';
+
+const visualDiffConfig = {
+	plugins: [
+		visualDiff()
+	],
+	reporters: [
+		defaultReporter(),
+		visualDiffReporter()
+	]
+};
+
+function getBrowsers() {
+	const browsers = ['chromium'];
+	if (argv.includes('firefox')) {
+		browsers.push('firefox');
+	}
+	if (argv.indexOf('webkit') > -1) {
+		browsers.push('webkit');
+	}
+	return browsers.map((b) => playwrightLauncher({
+		product: b,
+		createBrowserContext: ({ browser }) => browser.newContext({ deviceScaleFactor: 2, reducedMotion: 'reduce' })
+	}));
+}
```

Comment:
I think we need to nail down exactly how we want this to work. Should we always be using `playwrightLauncher` (for unit, axe, and vdiff)? Should we always include `chromium` if only `firefox` was passed in?

Reply:
I do think we always want reduced motion enabled... so yeah if we can always go through something like this that would be ideal.